apps/dms-material/src/styles.scss

```
@@ -242,3 +242,34 @@ textarea:focus-visible,
 .month-select-panel {
   min-width: 12rem !important;
 }
+
+// Epic 101, Story 101.3 — CDK virtual scroll content-wrapper containment fix.
+//
```

Stylelint (17.10.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)